scripts/continuous/run.sh

```diff
@@ -102,6 +103,13 @@ run_check() {
 			ret=0
 		fi
 	done
+
+	if [ "x$CONTINIOUS_SUCCESS_START_PATTERN" != "x" ] ; then
```

Comment:
I'm not sure about this part: you could just include `QVNCServer created` into the list. Or refactor a line above to 
```
SUCESS_PATTERNS=(
^run: success auto poweroff' 'embox>' '[a-z]\+@embox'
)
if [ "x$CONTINIOUS_SUCCESS_START_PATTERN" != "x" ]
    SUCESS_PATTERNS += '$CONTINIOUS_SUCCESS_START_PATTERN'
fi
for success_pattern in "$CONTINIOUS_SUCCESS_START_PATTERN[@]"; do 
```
(code excerpt is untested)

Reply:
Initially, I included `QVNCServer created` in the list, but then decided to use a bit more flexible way, since it's could be a common workaround for all applications running in kernel task.

But probably it's better to solve it simpler - just to include `QVNCServer created` in the list back like you propose.

Reply:
The current version is just feels like a workaround. Probably the whole success_pattern thing is wrong, as it require simple runner script to know all positive and negative outcomes. 
Can we ignore the error from qemu_bg and check for `QVNCServer created` in qt-vnc test script?

Reply:
I made it somehow by [this](https://github.com/embox/embox/pull/1277/commits/cdb95401bb1a9af21e0f5ea7f91713046150a6dd) and [this](https://github.com/embox/embox/pull/1277/commits/cbb035c53f4f122a7e6075fb8d9b8527a9dc3d4f).